classes/music_album.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './item'
+
+class MusicAlbum < Item
+  attr_reader :album_name, :artist_name, :publish_date, :on_spotify
+
+  def initialize(album_name, artist_name, publish_date, on_spotify)
+    super(publish_date)
```

Comment:
- [ ] Archived property accessed from item class
```suggestion
    super(publish_date, archived: archived)
```

Reply:
@RWUBAKWANAYO 
The required changes has been made